vegbilder-client/src/components/ImageView/ImageControlBar/ImageControlButtons.tsx

```diff
@@ -240,6 +274,23 @@ const ImageControlButtons = ({
     );
   };
 
+  const zoomInOut360Button = (zoomType: zoomType) => {
+    return (
+      <Tooltip title={zoomType === 'zoomIn' ? 'Zoom inn' : 'Zoom ut'}>
+        <IconButton
+          disabled={isHistoryMode || (zoomType === 'zoomIn' && isMinOrMaxZoom.isMaxZoom) || (zoomType === 'zoomOut' && isMinOrMaxZoom.isMinZoom)}
+          aria-label={zoomType}
+          className={isHistoryMode || (zoomType === 'zoomIn' && isMinOrMaxZoom.isMaxZoom) || (zoomType === 'zoomOut' && isMinOrMaxZoom.isMinZoom) ? classes.buttonDisabled : classes.button}
+          onClick={() => {
+            zoom360(zoomType);
+          }}
+        >
+          {zoomType === "zoomIn" ? <ZoomInIcon/> : <ZoomOutIcon/>}
+        </IconButton>
+      </Tooltip>
+    );
+  };
+
```

Comment:
Ville egentlig bruke zoom-komponenten som allerede eksisterte, men følte det ble veldig mange "if". Lagde derfor en ny komponent for 360, men er usikker på om den er god praksis.